my_model/template/cpp/my_model.cpp

```diff
@@ -13,10 +13,75 @@
 
 namespace mya {
 
-MyModel::MyModel() {}
+// Aerosol state specific to this model
+class MyState : public aero::State {
+public:
+  // (All data in the state is public for simplicity.)
+  std::vector<aero::Real> od,      // aerosol optical depth [m]
+                          od_ssa,  // aerosol single scattering albedo [-]
+                          od_asym; // aerosol asymmetric scattering optical
+                                   // depth [m]
```

Comment:
Why not private data members and friend class `aero::Model`?

Reply:
Considering that this class is not defined outside `my_model.cpp`, it's not available to anyone but `aero::MyModel`. :-)

Reply:
ah! ok, I missed that